Source/santactl/Commands/SNTCommandFileInfo.m

```diff
@@ -421,13 +421,10 @@ + (void)runWithArguments:(NSArray *)arguments daemonConnection:(SNTXPCConnection
 
   [filePaths enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
     NSBlockOperation *hashOperation = [NSBlockOperation blockOperationWithBlock:^{
-      if (isatty(STDOUT_FILENO) && !jsonOutput) {
-        printf("\rCalculating %lu/%lu", ++hashed, filePaths.count);
-      }
+      if (PrettyOutput()) printf("\rCalculating %lu/%lu", ++hashed, filePaths.count);
```

Comment:
Since this function would only be used in two places, leaving the PrettyOutput() check.